rules/Renaming/Rector/Class_/RenameAttributeRector.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Renaming\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Name\FullyQualified;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Property;
+use Rector\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Rector\AbstractRector;
+use Rector\Renaming\ValueObject\RenameAttribute;
+use Rector\ValueObject\PhpVersionFeature;
+use Rector\VersionBonding\Contract\MinPhpVersionInterface;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @see \Rector\Tests\Renaming\Rector\Class_\RenameAttributeRector\RenameAttributeRectorTest
+ */
+final class RenameAttributeRector extends AbstractRector implements ConfigurableRectorInterface, MinPhpVersionInterface
+{
+    /**
+     * @var RenameAttribute[]
+     */
+    private array $renameAttributes = [];
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Rename attribute class names',
+            [
+                new ConfiguredCodeSample(
+                    <<<'CODE_SAMPLE'
+#[SimpleRoute()]
+class SomeClass
+{
+}
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+#[BasicRoute()]
+class SomeClass
+{
+}
+CODE_SAMPLE
+                    ,
+                    []
```

Comment:
configurable code example should have entry example

Reply:
Thanks :+1:  Fixed in next commit